javascript/javascript1/week1/homework of  JSweek1/index.js

```diff
@@ -0,0 +1,49 @@
+//Age-ify ( A future age calculator)
+const yearOfBirth = 1991;
+const yearFuture = 2063;
+const age = yearFuture - yearOfBirth ;
+console.log("You will be " + age + " years old in " + yearFuture)
+
+//Goodboy-Oldboy(a DOG AGE CALCULATOR)
+const dogYearOfBirth = 2017;
+const  dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = dogYearFuture - dogYearOfBirth;
```

Comment:
why do you use the number for the shouldShowResultInDogYears variable?

Reply:
I'm sorry I don't get what you mean, but my I wrote it, so if you change the value of  dogYearOfBirth variable or  dogYearFuture  variable so it gives a different result for  shouldShowResultInDogYears variable.